ui/org.eclipse.pde.core/src/org/eclipse/pde/core/plugin/PluginRegistry.java

```diff
@@ -117,13 +118,29 @@ public static IPluginModelBase findModel(IProject project) {
 	/**
 	 * Returns a plug-in model associated with the given bundle description
 	 *
-	 * @param desc the bundle description
+	 * @param desc
+	 *            the bundle description
 	 *
-	 * @return a plug-in model associated with the given bundle description or <code>null</code>
-	 * 			if none exists
+	 * @return a plug-in model associated with the given bundle description or
+	 *         <code>null</code> if none exists
```

Comment:
Please also add a removal notice in the Java-doc, this allows JDT migrate existing callers via clean-ups.
```suggestion
	 *         <code>null</code> if none exists
	 * @deprecated Instead use {@link #findModel(Resource)}
```